docs/snippets/react/button-story-decorator.ts-4-9.mdx

```diff
@@ -0,0 +1,30 @@
+```tsx
+// Button.stories.ts|tsx
+
+import type { StoryObj, Meta, StoryFn } from '@storybook/react';
+
+import { Button } from './Button';
+
+const meta = {
+  /* 👇 The title prop is optional.
+   * See https://storybook.js.org/docs/7.0/react/configure/overview#configure-story-loading
+   * to learn how to generate automatic titles
+   */
```

Comment:
This would save a little bit of vertical space. @jonniebigodes WDYT, do you have a preference for either?

```suggestion
  /* 👇 The title prop is optional; To learn how to generate automatic titles see:
   * https://storybook.js.org/docs/7.0/react/configure/overview#configure-story-loading
   */
```

Reply:
But I see we use the same pattern in other message, so ignore. Consistency is better.